services/collectionsService.ts

```diff
@@ -0,0 +1,34 @@
+import { zdk } from '@shared'
+import { collectionAddresses } from 'constants/collection-addresses'
+import { CollectionSortKey, SortDirection } from '@zoralabs/zdk/dist/queries/queries-sdk'
+import * as Sentry from '@sentry/react'
+
+export async function collectionsService() {
+  try {
+    const data = await zdk.collections({
+      where: { collectionAddresses: collectionAddresses },
+      sort: { sortDirection: SortDirection.Asc, sortKey: CollectionSortKey.None },
+    })
+
+    const collections = data.collections
+
+    return {
+      props: {
+        fallback: collections?.nodes,
```

Comment:
I feel like fallback isn't descriptive enough. Something like initialCollections would do a better job of clarifying what data is being passed.